src/RecoveryServices/RecoveryServices.SiteRecovery/NetworkInterfaceCard/NewAzureRmAsrVmNicConfig.cs

```diff
@@ -363,5 +387,23 @@ private bool ValidateAndPopulateIPConfigs(ASRVMNicDetails vmNic)
 
             return true;
         }
+
+        PSIPConfigInputDetails ConvertToPSIPConfig(IPConfigDetails ipConfig)
```

Comment:
or you could write a mapper - check AutoMapper

Reply:
will stick with current approach as i see no performance benefit.